src/booleans.js

```diff
@@ -1,62 +1,100 @@
 function negate(a) {
   // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
   // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
   // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
   // your code here
-};
+  return !a && !b;
+}
 
 function one(a, b) {
   // your code here
-};
+  return (a && !b) || (!a && b);
+}
 
 function truthiness(a) {
   // your code here
-};
+  if (a === 0 || Number.isNaN(a) || a === null || a === undefined || a === '') {
+    return false;
+  }
+
+  return true;
```

Comment:
This is quite overengineered, you dont need to supply an exhaustive list of things that are falsy when its built into the language, have a look at https://developer.mozilla.org/en-US/docs/Glossary/Falsy for examples.
A cleaner way is to simply pass the value into the Boolean constructor and return that:
```suggestion
  return Boolean(a);
```